vmm/src/vm.rs

```diff
@@ -2472,6 +2472,7 @@ impl Pausable for Vm {
         self.vm
             .pause()
             .map_err(|e| MigratableError::Pause(anyhow!("Could not pause the VM: {}", e)))?;
+        info!("VM has been paused");
```

Comment:
Let's drop this change, since this is not related to the topic.

Reply:
> Let's drop this change, since this is not related to the topic.

This is to estimate downtime through logging. It may not be needed here, I will remove it.